src/components/twilio-voice-monitoring/routes.js

```diff
@@ -17,6 +17,10 @@ const {
 const client = Twilio(apiKeySid, apiKeySecret, { accountSid });
 const componentUrl = 'twilio-voice-monitoring';
 let callerCallSid;
```

Comment:
We don't need this anymore since we have the promise that contains the same info.